src/components/EmbeddedNavSection.vue

```diff
@@ -1,5 +1,12 @@
 <template>
   <nav :aria-label="$t('header.aria.primary')" class="navbar embedded">
+    <NuxtLink to="/" style="align-self: center; line-height: 0">
+      <img
+        alt="Openverse logo mark"
+        src="~/assets/logo.svg?data"
+        style="height: 24px; padding-right: 24px; width: auto"
+      />
```

Comment:
```suggestion
     <img
        alt="Openverse logo mark"
        src="~/assets/logo.svg?data"
        style="padding-right: 24px"
        width="160"
        height="24"
     />
```

Reply:
Setting width and height for images is supposed to prevent layout shifts. The logo actually doesn't cause any, but still...